beat/plotting.py

```diff
@@ -2351,7 +2373,28 @@ def draw_fuzzy_beachball(problem, po):
 
         beachball.plot_fuzzy_beachball_mpl_pixmap(
             m6s, axes, best_mt=best_mt, best_color='red', **kwargs)
-
+        ##Mahdi
+        if 'polarity' in problem.config.problem_config.datatypes:
+            (syn_poldatasets, syn_polarities), (obs_poldatasets) = extract_polarity_data(problem, po)
```

Comment:
whats the difference between syn_poldatasets and syn_polarities? why do you return it? in the end you use only obs_poldatasets

Reply:
at first, I made a mistake :)